.circleci/config.yml

```diff
@@ -325,10 +325,7 @@ workflows:
   nightly-integration-tests:
     triggers:
       - schedule:
+          <<: *filter-dev-branch
           cron: "0 13 * * *"
-          filters:
-            branches:
-              only: dev
     jobs:
       - integration-tests-alpha
-      - integration-tests-staging
```

Comment:
Should we keep the tests against staging in nightly? Is this meant to run tests after updating the staging backend?

Reply:
There was a previous discussion to remove it from nightly because we run tests against Staging API on PR commits and against Staging UI + API when merges.